integration/helpers/vite-template/vite.config.ts

```diff
@@ -3,5 +3,8 @@ import { defineConfig } from "vite";
 import tsconfigPaths from "vite-tsconfig-paths";
 
 export default defineConfig({
+  optimizeDeps: {
+    include: ["react", "react-dom"],
+  },
```

Comment:
I added this in hopes of getting rid of the additional asset preloads noted below - but if it's not helpful we can remove this change?